docs/source/TetraOpt_Algorithm/Parameters_and_Hyperparameters.md

```diff
@@ -19,4 +19,43 @@ Every problem will have a problem dimensionality determined by the domain of the
 |`precision` | string  | "float64" or "float32"                                                                                                                                                                                              | 
 |`point` | string  | "float64" or "float32"                                                                                                                                                                                              | 
 
+## Example
+```python
+from tq42.client import TQ42Client
+from tq42.experiment_run import ExperimentRun
+from tq42.compute import HardwareProto
+from tq42.algorithm import AlgorithmProto
 
+client = TQ42Client()
+client.login()
+
+parameters = {
+        "parameters": {
```

Comment:
why do use a dict here instead of a message like you were doing in all other examples?

Reply:
done